modules/huawei/include/hw_ivs.h

```diff
@@ -3,6 +3,9 @@
 
 #include "vms_interface.h"
 
+#include "windows.h"
```

Comment:
This will failed to compile in Linux. Use 

```
#ifndef _WIN32
#include "windows.h"
#endif // _WIN32
```

Reply:
Just knew about it. Why it is `#ifndef`? My understanding thinks it's `#ifdef`(?)
Compiler will consider the piece of code under those `#ifndef` if the compilation constant is not defined and `_WIN32` is a Windows system defined macro.

Reply:
My bad. Yes, it should be `#ifdef`.